CHANGELOG.md

```diff
@@ -44,6 +44,7 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ### Changed
 
+- #859 - added color diagnostic output and error messages.
```

Comment:
oops! Missed this too, wrong section for changelog, should be moved to unreleased